omnibus/config/software/datadog-agent-integrations.rb

```diff
@@ -88,35 +128,27 @@
       # We don't have auto_conf on windows yet
       if os != 'windows'
         if File.exist? "#{check_dir}/auto_conf.yaml"
-          mkdir check_conf_dir
-          copy "#{check_dir}/auto_conf.yaml", "#{check_conf_dir}/" unless File.exist? "#{check_conf_dir}/auto_conf.yaml"
+          mkdir check_conf_dir unless File.exists? (check_conf_dir)
+          copy "#{check_dir}/auto_conf.yaml", "#{check_conf_dir}/"
         end
       end
 
-      next unless File.exist?("#{check_dir}/requirements.txt") && !manifest['use_omnibus_reqs']
-
-      reqs = File.open("#{check_dir}/requirements.txt", 'r').read
-      reqs.each_line do |line|
-        all_reqs_file.puts line if line[0] != '#'
+      File.file?("#{check_dir}/setup.py") || next
+      if windows?
+        command "#{windows_safe_path(install_dir)}\\embedded\\scripts\\pip.exe #{build_args}", :cwd => "#{project_dir}\\#{check}"
```

Comment:
I think it's worth repeating the assignment `build_args = "wheel --no-deps ."` right before this command :)